ibis/backends/base/sqlglot/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
+
+import ibis.expr.operations as ops
+
+
+def unalias(op: ops.Value) -> ops.Value:
```

Comment:
I have mixed feelings about this.

It solves a problem which is that certain expressions cannot have aliases, despite reuse of the expression.

Here's an example:

```
select key as my_key
from group by key as my_key -- not allowed, but in ibis these are the same expression
```

so we have to compile the **unaliased** group by expressions in the `GROUP BY`, and the aliased ones in the `SELECT`.

Reply:
I believe this **must** be handled by compilers and doesn't indicate any design problems with ibis's IR.